Dockerfile.assisted-service-build

```diff
@@ -2,7 +2,10 @@ FROM golang:1.14.3
 
 ENV GO111MODULE=on
 
-RUN apt-get update && apt-get install -y docker.io libvirt-clients awscli python3-pip postgresql \
+RUN echo 'deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Debian_10/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list && \
+    curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Debian_10/Release.key | apt-key add -
+
+RUN apt-get update && apt-get install -y docker.io podman libvirt-clients awscli python3-pip postgresql \
```

Comment:
Running podman inside the docker container can be complicated to get working. Can the podman commands be run outside of skipper? Is podman installed on the Jenkins workers?